### PR TITLE
[16.0][IMP] mail_gateway: Allow to use expand on Gateway messages

### PR DESCRIPTION
**Pull request by chienandalu**
extending #1443 

Just added dummy compatibility with `mail_attach_existing_attachment` and `mail_composer_cc_bcc`

TT51108 cc @Tecnativa 

please check @etobella @pedrobaeza

### CODE REVIEW

**Comment by chienandalu** [comment]
Now its green... although the fix isn't very nice....

File: mail_gateway/models/res_partner.py [hunk 1]
```diff
@@ -66,6 +66,23 @@ class ResPartnerGatewayChannel(models.Model):
         "res.company", related="gateway_id.company_id", store=True
     )
 
+    def name_get(self):
```
Comment (pedrobaeza) [inline mail_gateway/models/res_partner.py:69]: Document why this method override.

**Comment by pedrobaeza** [comment]
The UI seems totally unordered:

imagen

Can you please fix it?

**Comment by chienandalu** [comment]
fixed

**Review by pedrobaeza — APPROVED** [review]
Working as expected. I merge as there's the implicit approval of @etobella:

/ocabot merge minor

**Pull request merged**